### ISSUE TITLE
Database and ConnectionPool aren't fully interchangeable

### ISSUE THREAD

**Issue by vribeiro1** [threatify/arango-orm#52]
Docs states that Database and ConnectionPool are interchangeable, but I experienced some problems moving from the first to the former. ConnectionPool doesn't have `has_graph` and `graph` methods. These are just examples of issues I had. Maybe it's missing more stuff I didn't check. Because of this I have problems instantiating Graph objects and managing my database graphs.

Am I missing something obvious when moving from one to the other? My code is like this:

With Database:

```
from arango import ArangoClient
from arango_orm import Database

client = ArangoClient(protocol=DB_PROTOCOL, host=DB_HOST, port=DB_PORT)
db = Database(client.db(DB_NAME, username=DB_USERNAME, password=DB_PASSWORD))
graph = MyGraph(connection=db)

if not db.has_graph(graph.__graph__):
    db.create_graph(graph)
```

With ConnectionPool:

```
from arango import ArangoClient
from arango_orm import ConnectionPool

clients = [ArangoClient(protocol=DB_PROTOCOL, host=DB_HOST, port=DB_PORT) for i in range(CONNECTION_POOL_SIZE)]
db = ConnectionPool(clients, dbname=DB_NAME, username=DB_USERNAME, password=DB_PASSWORD)
graph = MyGraph(connection=db)

if not db.has_graph(graph.__graph__):
    db.create_graph(graph)
```

**Comment by kashifpk** [comment]
Is this happening using the current master branch? ConnectionPool may be missing some stuff but calls like create_graph are also part of connection pool objects.

https://github.com/threatify/arango-orm/blob/master/arango_orm/connection_pool.py#L81

**Comment by vribeiro1** [comment]
It's happening with the latest version available in PyPI (arango-orm==0.5.1)

**Comment by kashifpk** [comment]
Sorry for the much delayed response. From python-arango 5.0 onwards, the ArangoClient class itself contains the hosts parameter so we no longer need to support this at the ORM level.
